python/servo/command_base.py

```diff
@@ -313,6 +327,9 @@ def build_env(self, gonk=False, hosts_file_path=None):
             if subprocess.call(['which', 'ld.gold'], stdout=PIPE, stderr=PIPE) == 0:
                 env['RUSTC'] = path.join(self.context.topdir, 'etc', 'rustc-with-gold')
 
+        # This sometimes ends up as unicode
+        env["PATH"] = env["PATH"].encode('ascii', 'ignore')
```

Comment:
Well, I guess I could encode as utf-8 here instead.  Sometimes, and I don't know why, the PATH value gets munged and turned into a unicode string.  That can't be passed in as an environment variable.  utf-8 is probably a better option here.